docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,103 @@
+
+**Important**: We encourage you to thoroughly review the license agreement and review all terms and conditions. Then accept to enable Einstein for Developers.
+
+Einstein for Developers is activated in the org. All the users in the org have access to the feature.
+
+### Add Users to the Org
+
+An admin can add additional users to the org as needed:
+
+1. From Setup, enter Users in the Quick Find box, then select **Users**.
+2. Click **New User** or **Add Multiple Users**.
+3. Select the appropriate license type and profile based on the user’s role.
+4. Select the **Generate passwords and notify user via email** checkbox.
+5. Click **Save**.
+   This procedure generates an email inviting the new users into the org.
+
+### Visual Studio Code Version
+
+VS Code releases a new version each month with new features and important bug fixes. You must be on VS Code Version 1.76 or higher to run the Einstein for Developers extension. You can manually check for updates from **Help** **> Check for Updates** on Linux and Windows or **Code > Check for Updates** on macOS.
+
+## Install Einstein for Developers Extensions
+
+Install Einstein for Developers:
+
+- If you’re using VS Code on your desktop, install [Einstein for Developers](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) from the Visual Studio Code marketplace.
+
+- If you are using Code Builder, click the Extensions icon in the Activity Bar in Code Builder, search for “Einstein for Developers” and click **Install**.
+
+**Note**: To use Einstein for Developers locally, you must have the [Salesforce Extensions pack]() installed in your VS Code for desktop application. See [Install Salesforce Extensions](https://developer.salesforce.com/tools/vscode/en/vscode-desktop/install) for more information.
+
+## Connect to an Org
+
+The Einstein for Developers tool is run in the context of a Salesforce org, in a Salesforce DX project. To use this tool:
+
+1. Go to **File** > **Open Folder** in the menu and open an existing Salesforce DX project in VS Code, or create one.
+
+2. Run the **SFDX: Authorize an Org** command to connect to the Salesforce sandbox org or scratch org that has Einstein for Developers enabled.
+
+The Einstein logo on the activity bar and in the status bar confirms that the extension is installed. Open the command palette and run `View: Show Einstein Developer Sidebar` to open the sidebar.
+
+![einstein installed](../../../images/einstein-installed.png)
+
+### Use Einstein for Developers in a Scratch Org
+
+Einstein for Developers is only available in scratch org editions that can author Apex:
+
+- Developer Edition
+- Enterprise Edition
+
+Activate Einstein for Developers by turning on the `EinsteinGPTForDevelopers` scratch org feature:
+
+1. Edit the `config/project-scratch-def.json` file in your DX project and add the “`EinsteinGPTForDevelopers`” feature to your existing feature list. For example:
```

Comment:
@sbudhirajadoc the scratch org changes look good - can we explicitly call out that the user will need to enable EGPT in the **dev hub** setup page for this to work? So the steps are:
1. enable in dev hub
2. log into dev hub in vs code
3. add feature to scratch org def file (could move this up in order)
4. create a scratch org from that dev hub

Reply:
fixed.